src/schemas/validation/schema.yaml

```diff
@@ -230,6 +230,52 @@ $defs:
         type: array
         items:
           $ref: '#/$defs/parameter-or-reference'
+      additionalOperations:
+        type: object
+        additionalProperties:
+          $ref: '#/$defs/operation'
+        not:
+          anyOf:
+            - properties:
+                get:
+                  $ref: '#/$defs/operation'
+              required:
+                - get
+            - properties:
+                put:
+                  $ref: '#/$defs/operation'
+              required:
+                - put
+            - properties:
+                post:
+                  $ref: '#/$defs/operation'
+              required:
+                - post
+            - properties:
+                delete:
+                  $ref: '#/$defs/operation'
+              required:
+                - delete
+            - properties:
+                options:
+                  $ref: '#/$defs/operation'
+              required:
+                - options
+            - properties:
+                head:
+                  $ref: '#/$defs/operation'
+              required:
+                - head
+            - properties:
+                patch:
+                  $ref: '#/$defs/operation'
+              required:
+                - patch
+            - properties:
+                trace:
+                  $ref: '#/$defs/operation'
+              required:
+                - trace
```

Comment:
This only catches additional methods in lowercase spelling.

Maybe @handrews or @karenetheridge know how to express this case-insensitively.